src/main/java/com/gary/backendv2/controller/EnumController.java

```diff
@@ -16,24 +15,24 @@
 @RequestMapping(path = "/enum")
 @RequiredArgsConstructor
 public class EnumController {
+	private static List<String> getEnumValues(Class<? extends Enum> cl) {
+		return Stream.of(cl.getEnumConstants())
+				.map(e -> e.name())
+				.collect(Collectors.toList());
+	}
```

Comment:
Przeniósł bym to do nowego pliku w` /src/main/java/com/gary/backendv2/utils
`
klasy kontrolerów powinny mieć tylko metody odpowiedzialne za przetwarzanie żądań http, a ta metoda to taki bardziej helper function który zwraca nam wszystkie wartości enumów, które może się przydadzą w innym miejscu w kodzie

Reply:
albo będziemy potrzebowali robić inne operacje na enumach to wtedy cały kod zarządzający nimi będzie bardziej czytelny i w jednym miejscu w repozytorium